data/community/members.yaml

```diff
@@ -0,0 +1,2125 @@
+technical-committee:
+  - name: jsuereth
```

Comment:
could we show name along with github alias?

Reply:
I would guess so, the name should be accessible via the github API

Reply:
from what I understand unfortunately that would need another API call per user, I will leave that out for now and use the github handles